src/argparcel/__init__.py

```diff
@@ -261,3 +261,18 @@ def parse[T: _typeshed.DataclassInstance](
     }
 
     return cls(**converted_kwargs)
+
+
+def uses_types[T](*types: type) -> Callable[[T], T]:
+    """Decorate a dataclass, and indicate types needed at runtime.
+
+    The existence of this method is a somewhat disgusting workaround to let you, the
+    user, declare to your static linting tools that a type is required at runtime, and
+    that the import must not be moved into a `TYPE_CHECKING` block.
+    """
+    del types
+
+    def _f[_T](cls: _T) -> _T:
+        return cls
+
+    return _f
```

Comment:
I'm not sure what it means for a local variable to be _private; could you please explain your reasoning here?

Reply:
1b353be